src/main/java/Duke.java

```diff
@@ -1,10 +1,109 @@
+import Exceptions.DukeException;
+import Storage.Storage;
+import Tasks.Deadline;
+import Tasks.Event;
+import Tasks.Task;
+import Tasks.Todo;
+import UI.Parser;
+import UI.Ui;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static void main(String[] args) throws DukeException {
+
+        //Initialising packages
+        Storage storage = new Storage();
+        Ui ui = new Ui();
+        Parser parser = new Parser();
+        List<Task> tasks = storage.loadTasks();
+
+        //Print welcome
+        ui.printWelcome();
+
+        //Execute Inputs
+        boolean isExit = false;
+        while (!isExit){
```

Comment:
```suggestion
        while (!isExit) {
```
Perhaps use whitespace before the open curly bracket (to make the code cleaner)?
I noticed the same issue in several other places too